bin/apisix

```diff
@@ -706,28 +706,43 @@ local function init_etcd(show_output)
     end
 
     local etcd_conf = yaml_conf.etcd
-    local uri = etcd_conf.host .. "/v2/keys" .. (etcd_conf.prefix or "")
 
     local timeout = etcd_conf.timeout or 3
+    local uri
+    local host_count = #(yaml_conf.etcd.host)
```

Comment:
Apache APISIX users may use the old `conf/config.yaml` and I think we should check the data type first.

For example Lua code:

```lua
if type(yaml_conf.etcd.host) == "string" then
    yaml_conf.etcd.host = {yaml_conf.etcd.host}
end

local host_count = #(yaml_conf.etcd.host)
...
```

Reply:
Yes that will be good

Reply:
@membphis  done